storage-api/src/test/org/apache/hadoop/hive/ql/exec/vector/expressions/TestStringExpr.java

```diff
@@ -49,6 +50,24 @@ public void test() throws Exception {
     assertEquals("Testing match at end of string", 24, find(pattern, input4));
   }
 
+  @Test
+  public void test2() throws Exception {
```

Comment:
Can you give a more meaningful name instead of `test2`?

And can you provide a qtest to make a end to end test?

Reply:
@zhangbutao I fixed test name and add qtest.

Reply:
Thanks. But i didn't reproduce the exception described in HIVE-26713 using your qtest on master branch.
Can you confirm that this issue exists on master?

Reply:
@zhangbutao Sorry, I fixed this because the qtest was missing some conditions.